lib/tasks/backfill_hashed_ssn_on_intake.rake

```diff
@@ -0,0 +1,19 @@
+namespace :backfill_hashed_ssn_on_intake do
```

Comment:
could be a [data migration](https://www.notion.so/cfa/Making-a-data-migration-bbd589c5f4df416bbe7d2aeda796cc39) instead of a rake task